server/events/project_command_runner.go

```diff
@@ -167,6 +167,7 @@ func (p *ProjectOutputWrapper) updateProjectPRStatus(commandName command.Name, c
 
 	// ensures we are differentiating between project level command and overall command
 	result := execute(ctx)
+	ctx.CommandResult = result
```

Comment:
```diff
+	ctx.CommandResult = result
```
This change is making the `TestProjectOutputWrapper` to fail

Reply:
@albertorm95 is this change needed?

Reply:
@albertorm95 it's odd that this line is causing the failure because it should not affect the `result` if statement on line 172. I imagine the code isn't falling into that if block because that is where `SetJobURLWithStatus` is called.

Perhaps it's a test issue?

Reply:
Yep, it looks like a test issue

https://github.com/runatlantis/atlantis/blob/9ca57d61897adabb4a844f61d9f2be95b5e2f480/server/events/project_command_runner_test.go#L230

https://github.com/runatlantis/atlantis/blob/9ca57d61897adabb4a844f61d9f2be95b5e2f480/server/events/models/commit_status.go#L22-L38

Basically, since you changed the status of the pr check, the test still thinks the status should be pending/success/failed.

Please update the test and it should work as expected.

cc: @albertorm95